keras/src/ops/nn.py

```diff
@@ -2918,9 +2918,12 @@ def _broadcast(v):
         # Calculate outputs with only variance and gamma if rms scaling
         # is enabled
         # Calculate the variance along self.axis (layer activations).
-        variance = backend.numpy.var(x, axis=axis, keepdims=True)
-        inv = backend.math.rsqrt(variance + epsilon)
-
+        inv = backend.math.rsqrt(
```

Comment:
maybe keep it two lines as before, make it a little easier to how epsilon is handled
```
ms = backend.numpy.mean(backend.numpy.square(x), axis=axis, keepdims=True)
inv = backend.math.rsqrt(ms + epsilon)
```